python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -44,14 +46,23 @@ class PhoSimRepackager:
     MEFs with one HDU per amp, using LSE-400 for header conformity.
     """
 
-    TELCODE = 'MC'  # Telescope Code: MC is Main Camera
-    CONTRLLR = 'H'  # Controller: H for pHosim
+    CONTRLLR = "H"  # Controller: H for pHosim
 
-    def __init__(self):
+    def __init__(self, prefix="lsst"):
 
-        self.camera = LsstCam().getCamera()  # obs_lsst mapper camera object
+        # Use appropriate obs_lsst mapper camera object
+        # and telescope code
+        if prefix == "lsst":
+            self.camera = LsstCam().getCamera()
+            self.telcode = "MC"  # Main Camera
+            self.instrument = "lsstCam"
 
-    def process_visit(self, visit_dir, out_dir=None, prefix='lsst', verbose=False):
+        elif prefix == "comcam":
+            self.camera = LsstComCam().getCamera()
+            self.telcode = "CC"  # Commissioning Camera
+            self.instrument = "comCam"
+
+    def process_visit(self, visit_dir, out_dir=None, prefix="lsst", verbose=False):
```

Comment:
Please update the doc strings of `out_dir`, `prefix`, and `verbose` to be "optional" with the default values.